services/geocoding.py

```diff
@@ -1,20 +1,23 @@
 import re
-
+import json
+import googlemaps
 from dataclasses import dataclass
 from typing import List, Mapping
-
 from returns.io import impure_safe
-
 from config.settings import Settings
 from models.point_of_interest import PointOfInterest
 
 
+
+
 @dataclass
 class Point:
     lat: str
     lng: str
 
-
+def make_geocode_request(address: str, gmaps) -> Point:
```

Comment:
I would recommend for `make_geocode_request` to do nothing other that IO, not even access `location` or construct the `Point` because it won't be covered by unit test so it should have as little code as possible.

Ideally it should return just the response:

```
return gmaps.geocode(address);
```